rxui-sample-java/src/test/java/com/artemzin/rxui/sample/java/TestMainView.java

```diff
@@ -13,12 +13,12 @@
 
     final PublishSubject<String> login = PublishSubject.create();
     final PublishSubject<String> password = PublishSubject.create();
-    final PublishSubject<Void> signInClicks = PublishSubject.create();
+    final PublishSubject<Object> signInClicks = PublishSubject.create();
 
-    Action1<Void> signInEnable = mock(Action1.class);
-    Action1<Void> signInDisable = mock(Action1.class);
-    Action1<AuthService.Success> signInSuccess = mock(Action1.class);
-    Action1<AuthService.Failure> signInFailure = mock(Action1.class);
+    Consumer<Object> signInEnable = mock(Consumer.class);
```

Comment:
Just curios why are you not using `@Mock` and then the Mockito Rule or `MockitoAnnotations.initMocks(this)`